.github/workflows/doc-sync-check.yml

```
@@ -0,0 +1,256 @@
+                } catch (error) {
+                  console.log(
+                    `API call attempt ${attempt} failed:`, error.message
+                  );
+
+                  // Check if it's a rate limit error
+                  // (status 403 with rate limit message)
+                  if (error.status === 403 &&
+                      error.message.includes('rate limit')) {
+                    const resetTime =
+                      error.response?.headers['x-ratelimit-reset'];
+                    const resetTimeMs = resetTime && 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)